docker/demisto-sdk/README.md

```diff
@@ -0,0 +1,54 @@
+# Demisto-SDK docker
+
+Run demisto-sdk commands from within a docker.
+
+You can use this image to run validations locally on your work environment or inside your ci.
+
+## The Content Repository
+
+To use the Demisto-SDK you should have a content-like repository (or use the official [content repository](https://github.com/demisto/content)).
+You can generate one from this [template](https://github.com/demisto/content-external-template)
+
+## Mounts
+
+We use volume mount in our environment to run on local content repository.
+Please note that there's a performance issue with mount on MacOS and Windows.
+You can solve the performance issue by running on Linux/Windows WSL2 or by cloning your repository to the `/content` directory inside the container.
```

Comment:
```suggestion
Please note that mounting on MacOS and Windows may cause slowness. 

To ensure the best performance, please either:
- Use a linux machine
- Use [Windows WSL2](https://docs.microsoft.com/en-us/windows/wsl/install) 
- Clone your content repository to the `/content` directory inside the container.
```

do we explain how to do the cloning?

Reply:
If someone is working with a docker, he should be familiar with vcs, container etc. I think we should draw a line between where we support and where we are not.